tests/core/transition/hermeticity_test.go

```diff
@@ -123,12 +210,15 @@ func assertDependsCleanlyOnWithFlags(t *testing.T, targetA, targetB string, flag
 	cqueryOut := string(bytes.TrimSpace(out))
 	configHashes := extractConfigHashes(t, cqueryOut)
 	if len(configHashes) != 1 {
-		t.Fatalf(
-			"%s depends on %s in multiple configs with these differences in rules_go options: %s",
-			targetA,
-			targetB,
-			strings.Join(getGoOptions(t, configHashes...), "\n"),
-		)
+		differingGoOptions := getGoOptions(t, configHashes...)
```

Comment:
Also, in `getGoOptions`, can you run `bazel config --output=json` and parse the json instead?